luigi/parameter.py

```diff
@@ -222,6 +257,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
+        """
+        Parse a list of values from the scheduler.
+
+        Only possible if this parameter has a batch method. This will combine the list into a single
+        parameter value using batch method.
+
+        :param xs: list of values to parse and combine
+        :return: the combined parsed values
+        """
+        if self._batch_method is None:
+            raise NotImplementedError('No batch method found')
+        elif not xs:
+            raise ValueError('Empty parameter list passed to parse_list')
+        else:
+            return self._batch_method(map(self.parse, xs))
```

Comment:
Ah. I see. I understand the issue.

If so. I suggest the following:

 * We start simple, lets not aim for support having lists yet as it's not as easy as one might think
 * Clearly say in the docs that `<func>` in `batch_method=<func>` must be a pure python function of signature `<func> :: [A] -> A`. (or however you would explain this in simple terms)
 * Also add a runtime check that checks that the type returned by `<func>` is correct.

Does that make sense @daveFNbuck?